plat/arm/board/common/board_arm_trusted_boot.c

```diff
@@ -1,13 +1,17 @@
 /*
- * Copyright (c) 2015-2016, ARM Limited and Contributors. All rights reserved.
+ * Copyright (c) 2015-2017, ARM Limited and Contributors. All rights reserved.
  *
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
 #include <arm_def.h>
 #include <assert.h>
 #include <platform.h>
+#if USE_TBBR_DEFS
+#include <tbbr_oid.h>
+#else
 #include <platform_oid.h>
+#endif
```

Comment:
I guess I wasn't clear enough. I was suggesting the platform make file (arm_common.mk for the ARM platforms) should set USE_TBBR_DEFS to 1 and use tbbr_oid.h unconditionally in the source files.